### DISCUSSION TITLE
nav or pages?

### DISCUSSION THREAD

**Discussion by balabalabalalaba** [mkdocs/mkdocs#2524] [General]
the ```mkdocs.yml``` should be:
```
nav:
    - Home: index.md
    - About: about.md
```
or : 
```
pages:
- [index.md, Home]
- [about.md, About]
```
?
the guide said the first but I can't achieve it. Is the guide too old?

**Comment by Andre601** [comment]
Should be nav:

Just make sure you use the latest version of mkdocs.
See https://github.com/Andre601/mkdocs-template for a detailed mkdocs.yml

**Reply by balabalabalalaba** [reply]
thanks! I used conda  to install mkdocs(which is 0.16.3) and I reinstall it with pip, it can work now.
